security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/MutualTlsKeycloakService.java

```diff
@@ -0,0 +1,71 @@
+package io.quarkus.ts.security.oidcclient.mtls;
+
+import static io.quarkus.test.utils.PropertiesUtils.RESOURCE_PREFIX;
+import static io.quarkus.test.utils.PropertiesUtils.SECRET_PREFIX;
+import static java.lang.String.format;
+
+import io.quarkus.test.bootstrap.KeycloakService;
+import io.quarkus.test.bootstrap.Protocol;
+
+public class MutualTlsKeycloakService extends KeycloakService {
```

Comment:
This implementation should be merged into QuarkusTestFramework [keycloakService](https://github.com/quarkus-qe/quarkus-test-framework/blob/main/quarkus-test-service-keycloak/src/main/java/io/quarkus/test/bootstrap/KeycloakService.java). Could you create an issue in [QuarkusTestFramework](https://github.com/quarkus-qe/quarkus-test-framework) in order to add this issue to the backlog.

Reply:
Shall do, let's merge this MR first and update it once I've moved `MutualTlsKeycloakService` to QuarkusTestFramework.